circle.yml

```diff
@@ -5,9 +5,9 @@ machine:
 dependencies:
   override:
     # Install the gruntwork-module-circleci-helpers and use it to configure the build environment and run tests.
-    - curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version 0.0.9
-    - gruntwork-install --module-name "gruntwork-module-circleci-helpers" --repo "https://github.com/gruntwork-io/module-ci" --tag "v0.0.10"
-    - configure-environment-for-gruntwork-module --terraform-version NONE --packer-version NONE --go-src-path .
+    - curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version 0.0.13
+    - gruntwork-install --module-name "gruntwork-module-circleci-helpers" --repo "https://github.com/gruntwork-io/module-ci" --tag "v0.0.18"
+    - configure-environment-for-gruntwork-module --packer-version NONE --go-src-path .
```

Comment:
The current test runs Terragrunt by [directly invoking the CLI app](https://github.com/gruntwork-io/terragrunt/pull/18/files#diff-e38680b417b49add3697d02eb94cf52fR39), but the point is well-taken. To avoid any confusion, I'll disable terragrunt explicitly.